php/libraries/NDB_BVL_InstrumentStatus.class.inc

```diff
@@ -201,6 +201,12 @@ class NDB_BVL_InstrumentStatus
             ['CommentID' => $this->_commentID]
         );
 
+        $GLOBALS['DB']->update(
```

Comment:
the flag table has `enum('In Progress','Complete') ` for Data_entry while the participant_accounts has `enum('Created','Sent','In Progress','Complete') `.

The instrument (flag) gets set to "In Progress" when it's accessed if the value is null, which will cause the participant_accounts value to get overwritten with "In Progress" if a data entry staff accesses the instrument after it's sent/created, even if the participant has never seen it.

I think this is only intended to be done if the value is "Complete"